crates/wasmtime/src/frame_info.rs

```diff
@@ -79,10 +70,6 @@ impl StoreFrameInfo {
         // may be a valid PC value
         let end = end - 1;
 
-        if self.contains_pc(start) {
-            return;
-        }
-
```

Comment:
Similarly confused with what's going on over here.

Reply:
This was ensuring we don't register the same module more than once in the store, however, this could probably just be `if self.module(start).is_some()` so it doesn't bother looking up the pc into the already registered module.

Reply:
Oh wait, I just caught up to the changes below.  Yeah, this shouldn't be necessary any more.